src/test/java/LoginTests.java

```diff
@@ -15,9 +15,15 @@ public static void LoginEmptyEmailPasswordTest() {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://testpro.io/";
+        String url = "https://bbb.testpro.io/";
         driver.get(url);
+
+
+
+
         Assert.assertEquals(driver.getCurrentUrl(), url);
         driver.quit();
+
```

Comment:
Test failed!

![image](https://user-images.githubusercontent.com/104761482/221347497-5d28a33e-21cb-4b72-8abd-fa107d00dc81.png)

**Hint**

**Step 1** 

Please move class HomeWork16 in package test - java

![image](https://user-images.githubusercontent.com/104761482/221347572-573f494e-1e4d-4602-864f-bb3897a9f96e.png)

![image](https://user-images.githubusercontent.com/104761482/221347605-e70dfc4a-33c1-4983-9532-276b88ece4cd.png)

**Step 2** 

java.lang.IllegalStateException: The path to the driver executable The path to the driver executable must be set by the webdriver.chrome.driver system property; 

Solution: **public class HomeWork16 extends BaseTest**